components/link/link.js

```diff
@@ -139,13 +156,31 @@ class Link extends FocusMixin(LitElement) {
 			'truncate': this.lines > 0
 		};
 		const styles = (this.lines > 0) ? { '-webkit-line-clamp': this.lines } : {};
+		if (this.newWindow && this.target === undefined) {
+			this.target = '_blank';
+		}
```

Comment:
Let's not actually modify the target property, since that's mutating something that isn't really this code's responsibility and will also trigger another render.

```suggestion
		const target = (this.newWindow && this.target === undefined) ? '_blank' : this.target;
```